src/__tests__/reducers/articleReducer.test.js

```diff
@@ -0,0 +1,154 @@
+import reducer from '../../redux/reducers/articleReducer';
+import * as articleTypes from '../../redux/actions-types/articleTypes';
+import { articleData } from '../../__mocks__/dummyData';
+
+import store from '../../redux/store';
```

Comment:
Why not use mocking store?

Reply:
@Nziranziza It is for testing the reducer, not the component

Reply:
@Nziranziza Your feedback has helped me changed it to the initial state instead of mocking. Thanks.